var/spack/repos/builtin/packages/darshan-runtime/package.py

```diff
@@ -21,6 +21,9 @@ class DarshanRuntime(Package):
     maintainers = ['shanedsnyder', 'carns']
 
     version('develop', branch='master')
+    version('3.2.1', sha256='d63048b7a3d1c4de939875943e3e7a2468a9034fcb68585edbc87f57f622e7f7')
+    version('3.2.0-pre1', sha256='afc11609420bab99a5f2dd7b4f4aa4c34f5423a5ca438c5f2d444dc3efe6dfa8')
```

Comment:
Ah, good call. Definitely should be sorted above 3.2.0, but you're right that there's not really any reason for users to choose the pre-release version over 3.2.0. I just removed it from both packages to avoid confusion.